notebooks/workflow_cost_estimator/main.py

```diff
@@ -48,9 +48,12 @@ def list_submissions_chronological():
     def cost_for_submission(submission_id: str):
         submission = workflows.get_submission(submission_id)
         for wf in submission['workflows']:
-            for shard_info in workflows.estimate_workflow_cost(submission_id, wf['workflowId']):
-                shard_info['workflow_id'] = wf['workflowId']
-                yield shard_info
+            try:
```

Comment:
Instead of the try:except: clause, lets use:
```
if 'workflowId' in wf:
    ...
else:
    print("No workflow id found for {useful_info_to_user}")
```

Is there information available in `wf` that would be useful to report to the user?

Reply:
You make a good point... I realized the workflow status is buried in there, so I was able to handle a few more edge cases. Thanks!